Source/PersonalityInsightsV3/PersonalityInsights.swift

```diff
@@ -0,0 +1,274 @@
+            let error = NSError(domain: domain, code: 0, userInfo: userInfo)
+            failure?(error)
+            return
+        }
+
+        getProfile(
+            version: version,
+            fromContent: content,
+            withType: "text/html",
+            acceptLanguage: acceptLanguage,
+            contentLanguage: contentLanguage,
+            rawScores: rawScores,
+            consumptionPreferences: consumptionPreferences,
+            failure: failure,
+            success: success
+        )
+    }
+
+    /**
+     Analyze input content items to generate a personality profile.
+     
+     - parameter version: The date in the form "YYYY-MM-DD" that will specify a version no later than the date provided.
+     - parameter contentItems: The content items to analyze.
+     - parameter acceptLanguage: The desired language of the response.
+     - parameter contentLanguage: The language of the text being analyzed.
+     - parameter rawScores: If true, then a raw score for each characteristic is returned in
+        addition to a normalized score. Raw scores are not compared with a sample population.
+        A raw sampling error for each characteristic is also returned.
+     - parameter consumptionPreferences: If true, then information inferred about consumption preferences is returned in addition to the results.
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed with the personality profile.
+     */
+    public func getProfile(
+        version: String,
+        fromContentItems contentItems: [ContentItem],
+        acceptLanguage: String? = nil,
+        contentLanguage: String? = nil,
+        rawScores: Bool? = nil,
+        consumptionPreferences: Bool? = nil,
+        failure: ((Error) -> Void)? = nil,
+        success: @escaping (Profile) -> Void)
+    {
+        let json = JSON(dictionary: ["contentItems": contentItems.map { $0.toJSONObject() }])
+        guard let content = try? json.serialize() else {
+            let failureReason = "Content items could not be serialized to JSON."
+            let userInfo = [NSLocalizedFailureReasonErrorKey: failureReason]
+            let error = NSError(domain: domain, code: 0, userInfo: userInfo)
+            failure?(error)
+            return
+        }
+
+        getProfile(
+            version: version,
+            fromContent: content,
+            withType: "application/json",
+            acceptLanguage: acceptLanguage,
+            contentLanguage: contentLanguage,
+            rawScores: rawScores,
+            consumptionPreferences: consumptionPreferences,
+            failure: failure,
+            success: success
+        )
+    }
+
+    /**
+     Analyze content to generate a personality profile.
+ 
+     - parameter version: The date in the form "YYYY-MM-DD" that will specify a version no later than the date provided.
+     - parameter content: The content to analyze.
```

Comment:
I think the comments should be updated to reflect the updated external names with Swift 3.0. This is probably something that is a holdover from PersonalityInsightsV2, but it would be good to update it in PersonalityInsightsV3.

In other words, the external parameter name has been changed to `fromContent` instead of `content`, so that should be updated here and throughout this file. It would be good to check the other parameters, too.